### ISSUE TITLE
Group chat icon symbols have a discrepancy with mobile

### ISSUE THREAD

**Issue by churik** [status-im/status-desktop#15230]
# Bug Report

## Description

According to [design](https://www.figma.com/design/7KIYbhoqNGAIFonE0w9TDz/Messages-for-Mobile?node-id=1424-342512&t=46Z51UmvwKkt5h8h-4) mobile group chat icon looks like :
![IMAGE 2024-06-18 11:48:13](https://github.com/status-im/status-desktop/assets/4557972/6f1309fa-9fec-4dc8-afb3-78eea488e541)

On desktop it is shown as 2-letters:
![Status Desktop 2024-06-18 11-50-51](https://github.com/status-im/status-desktop/assets/4557972/953de6dd-8c0e-43d3-bb39-3ff3d8ced918)

## Steps to reproduce

1. Create a group chat with no photo and compare icons on desktop and mobile

#### Expected behavior
as per design (matches mobile) Link to design: https://www.figma.com/design/IBmFKgGL1B4GzqD8LQTw6n/Design-System-for-Desktop%2FWeb?node-id=14584-169312&t=2k2qvjLmTblJMAGq-4

#### Actual behavior
shown similarly to 1-1 chats

### Additional Information

- Status desktop version:
- Operating System:

**Comment by caybro** [comment]
CC @noeliaSD seems like a rather easy fix

**Comment by jrainville** [comment]
We are still on old designs, so I don't know if it makes sense to match on certain areas and not others. We can loop @benjthayer on it.

**Comment by noeliaSD** [comment]
Maybe @xAlisher can also answer while `Ben` is not available!

**Comment by noeliaSD** [comment]
I add it to the milestone but low priority!

**Comment by churik** [comment]
@caybro also it is a regression from the fix for 1-1 chat https://github.com/status-im/status-desktop/issues/14667

**Comment by caybro** [comment]
@churik so if I have 10 group chats, they should all have the same icon?

**Comment by caybro** [comment]
Ping @xAlisher: should we follow mobile design here, or stay as-is (2 letter identicons for group chats)?

**Comment by noeliaSD** [comment]
There are still some details to be confiremed. Moved to next milestone bc of team capacity.

**Comment by caybro** [comment]
Ping @benjthayer ; any directions we should take here?

**Comment by benjthayer** [comment]
> Ping @benjthayer ; any directions we should take here?

If it is a regression as @churik states then we should review what was previously implemented for this. Perhaps we can do that first?

If no alignment was implemented we could look to using the desktop equivalent icon for group chats so desktop is aligned with mobile. While this means the same icon for all group chats, I believe the colour for the chat is randomly generated and the user can always edit the group image to make identifying the correct chat easier. 

Are there any other considerations from a technical or experience perspective I'm missing?

**Comment by caybro** [comment]
> > Ping @benjthayer ; any directions we should take here?
> 
> If it is a regression as @churik states then we should review what was previously implemented for this. Perhaps we can do that first?
> 
> If no alignment was implemented we could look to using the desktop equivalent icon for group chats so desktop is aligned with mobile. While this means the same icon for all group chats, I believe the colour for the chat is randomly generated and the user can always edit the group image to make identifying the correct chat easier.
> 
> Are there any other considerations from a technical or experience perspective I'm missing?

It's not a regression and haven't seen this identical group chat image being used in any current desktop designs; the question here really is should we now follow mobile design? That direction seems to be very adhoc, following mobile design in some areas only. Imho this should come as part of the whole desktop redesign

**Comment by benjthayer** [comment]
> > > Ping @benjthayer ; any directions we should take here?
> > 
> > 
> > If it is a regression as @churik states then we should review what was previously implemented for this. Perhaps we can do that first?
> > If no alignment was implemented we could look to using the desktop equivalent icon for group chats so desktop is aligned with mobile. While this means the same icon for all group chats, I believe the colour for the chat is randomly generated and the user can always edit the group image to make identifying the correct chat easier.
> > Are there any other considerations from a technical or experience perspective I'm missing?
> 
> It's not a regression and haven't seen this identical group chat image being used in any current desktop designs; the question here really is should we now follow mobile design? That direction seems to be very adhoc, following mobile design in some areas only. Imho this should come as part of the whole desktop redesign

Yeah if its not a regression, let's pause this one and look at mobile alignment more holistically when higher priority items are covered off.
